src/gitops/Statuses/HealthStatus.test.tsx

```
@@ -4,33 +4,33 @@ import HealthStatus, { HealthStatusIcon } from './HealthStatus';
 describe('HealthStatus', () => {
   it('renders Healthy', () => {
     expect(renderToStaticMarkup(<HealthStatus status="Healthy" />)).toMatchInlineSnapshot(
-      `"<div><div><svg data-icon="HeartIcon" style="color:var(--pf-v5-global--success-color--100)"></svg> Healthy</div></div>"`,
+      `"<div><div><svg data-icon="HeartIcon" style="color:var(--pf-t--global--icon--color--status--success--default)"></svg> Healthy</div></div>"`,
     );
   });
 
   it('renders Degraded', () => {
     expect(renderToStaticMarkup(<HealthStatus status="Degraded" />)).toMatchInlineSnapshot(
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)